src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -447,12 +497,27 @@ public static String convertToStandardDuration(Long timeInMilliseconds) {
     }
 
     /**
-     * Parses a {@code LocalDateTime} object from a date time string according to a pattern.
-     * Returns {@code null} on error.
+     * Parses an {@code Instant} object from a datetime string in the {@link SystemParams#DEFAULT_DATE_TIME_FORMAT}.
+     *
+     * @param dateTimeString should be in the format {@link SystemParams#DEFAULT_DATE_TIME_FORMAT}
+     * @return the parsed {@code Instant} object, or {@code null} if there are errors
+     */
+    public static Instant parseInstant(String dateTimeString) {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(SystemParams.DEFAULT_DATE_TIME_FORMAT);
+        try {
+            return ZonedDateTime.parse(dateTimeString, formatter).toInstant();
+        } catch (DateTimeParseException e) {
+            Assumption.fail("Date in String is in wrong format.");
+            return null;
+        }
+    }
+
+    /**
+     * Parses a {@code LocalDateTime} object from a datetime string and parsing pattern.
      *
-     * @param dateTimeString    the string containing the date and time
-     * @param pattern           the pattern of the date and time string
-     * @return                  the parsed {@code LocalDateTime} object, or {@code null} if there are errors
+     * @param dateTimeString the string containing the datetime
+     * @param pattern        the parsing pattern of the datetime string
+     * @return the parsed {@code LocalDateTime} object, or {@code null} if there are errors
      */
     public static LocalDateTime parseLocalDateTime(String dateTimeString, String pattern) {
```

Comment:
Is it a bit inconsistent to not remove the word `Local` here? (and in `parseLocalDate` as well)

Reply:
I prefer to leave the `Local` in because this method returns a `java.time` object. This is similar to the case of `formatLocalDateTime` and `formatInstant`, even though their callers are all of the form `formatDate...` or `formatDateTime...`, in order to show that they are the main, general purpose methods that their facades call. The only difference here is that there aren't any facades for `parseLocalDateTime`, but I think it's important to leave it inside `TimeHelper` in case the need arises in the future.